### DISCUSSION TITLE
Bug in Apple’s Availability - Apps intended for specific regions are being made accessible in all 175 countries by Apple.

### DISCUSSION THREAD

**Discussion by kpowelltech** [fastlane/fastlane#21623] [Q&A]
We’re encountering an issue where the previously defined territorial scopes are being wiped out upon Fastlane’s submission of the app to App Store Connect. Consequently, the app becomes accessible in all 175 countries. Given that our team is responsible for updating many iOS apps for various global merchants, this bug poses a significant problem. We’re currently debugging to ascertain whether the issue originates from our CI/CD or Fastlane setup.”

**Comment by fabioalmeida** [comment]
I'm having the same issue, my app Availability is being reset to all 175 countries with every execution of `fastlane deliver`. Any workaround for this?

**Comment by kpowelltech** [comment]
Hello @fabioalmeida, just a quick update: Although our apps are free, we noticed that setting the price_tier to 0 during the Deliver step might have caused Fastlane to categorize them as paid apps.

Since removing the price_tier code from the Deliver process, our apps maintain their original territories without being overwritten.
